CMakeLists.txt

```diff
@@ -181,9 +193,10 @@ else ()
     endif()
 endif()
 
+option(SUPRESS_WARNINGS "Silence all warnings" OFF)
+
 if (NOT MSVC)
     add_compile_options( -g)
-    add_compile_options( -pthread )
```

Comment:
As above, I don't think this change is needed right now.

Reply:
Okay, will revert.